lib/Service/RowService.php

```diff
@@ -66,6 +67,20 @@ public function formatRows(array $rows): array {
 		return array_map(fn (Row2 $row) => $row->jsonSerialize(), $rows);
 	}
 
+	/**
+	 * @param Row2[] $rows
+	 * @return TablesPublicRow[]
+	 */
+	public function formatRowsForPublicShare(array $rows): array {
+		return array_map(static function (Row2 $row): array {
+			$rowData = $row->jsonSerialize();
+			unset($rowData['tableId']);
```

Comment:
Why do we unset the `tableId` but not the column/row ids? What makes tableId more riskier